watertap/data/techno_economic/component_list.yaml

```diff
@@ -78,6 +82,8 @@ perfluorootanoate_pfoa:
   constituent_longform: Perfluorootanoate (PFOA)
 phosphates:
   constituent_longform: Phosphates
+phosphate_as_phosphorus:
```

Comment:
I went with our original intent of being inclusive rather than exclusive. For the phosphorus capture unit, I have a conditional that makes sure both are not specified.